frontend/src/api/EdnaApi.ts

```diff
@@ -81,7 +81,7 @@ export default function EdnaApi(axios: AxiosInstance): EdnaApiInterface {
           // TODO remove this mock transformation
           const byCompound = groupCompounds(response.data);
           return Object.entries(byCompound).map(([cmpId, measurements]) => ({
-            target: "target",
+            target: "",
```

Comment:
Why?

Reply:
because it's just dummy thing, which I will remove after all. for now it's "" because compoundId from old upload method already contains target and I wanted to have proper graph title